.github/workflows/release-go-master.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Release master branch
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #, macos-latest, windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.14.x'
```

Comment:
so far we have version `1.13` in our `go.mod` file. We should keep this in synch